src/codeu/chat/client/core/UserContext.java

```diff
@@ -35,6 +35,26 @@ public UserContext(User user, BasicView view, BasicController controller) {
     this.controller = controller;
   }
 
+  public String statusUpdate() {
+    return controller.newStatusUpdate(user.id);
+  }
+
+  public void followUser(User userB) {
+    controller.followUser(user, userB);
+  }
+
+  public void unfollowUser(User userB) {
+    controller.unfollowUser(user, userB);
+  }
+
+  public void unfollowConversation(Uuid conversation) {
```

Comment:
Be consistent.

Personally, I would have the functions here in UserContext take Object(User, Conversation).  I would have the controller functions also take Objects(User, Conversation), but then for the serializer I would send only the UUID of each.